### PR TITLE
docs: WIP: SSO section

### PR DESCRIPTION
**Pull request by ragog**

### CODE REVIEW

File: site/content/docs/single-sign-on/_index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (ragog) [inline site/content/docs/single-sign-on/_index.md:1]: @ianaya89 do you know why this page is not showing on the sidebar under "Single sign-on", in the docs? Played around with it but couldn't figure it out yet.

File: site/content/docs/single-sign-on/microsoft-azure-ad.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- Microsoft Azure AD Domain (e.g. company.com)
+- [Client ID](https://auth0.com/docs/connections/enterprise/azure-active-directory)
+- Client Secret
+
+## Testing the SSO integration
+
+After configuration has taken place on Checkly's side, you will receive confirmation via e-mail. Once that has happened, you should be able to log in to Checkly via SSO already. Entering an email address associated with the domain you have provided in the login prompt should result in the password field disappearing:
+
+![checkly login prompt without password screenshot](/docs/images/single-sign-on/checkly-login-prompt-sso.png)
```
Comment (tnolet) [inline site/content/docs/single-sign-on/microsoft-azure-ad.md:23]: This image seems to be broken

**Review by tnolet — CHANGES_REQUESTED** [review]
Looks & reads great. Just one niggle with a broken image

**Review by tnolet — APPROVED** [review]
LGTM

**Pull request merged**